src/runtime-rs/config/configuration-qemu-runtime-rs.toml.in

```diff
@@ -392,6 +392,13 @@ pflashes = []
 # Default 0
 #pcie_root_port = 2
 
+# Before hot plugging a PCIe device onto a switch port, you need add a pcie_switch_port device fist.
+# Use this parameter when using some large PCI bar devices, such as Nvidia GPU
+# The value means how many devices attached onto pcie_switch_port will be created.
+# This value is valid when hotplug_vfio_on_root_bus is true, and machine_type is "q35"
+# Default 0
+#pcie_switch_port = 2
```

Comment:
Seeing as it's illegal to have to both `pcie_root_port` and `pcie_switch_port` set to non-zero values, shouldn't we document this limitation for our users?

Reply:
I based my comment on the final hunk of the final commit of this series which seems to cause the config file validation to fail if both are non-zero.  Unfortunately I know too little about actual PCIe topologies to decide whether that's fair or not. :-)

Reply:
I've approved already but I'd like to clarify - the check making it illegal to have both `pcie_root_port` and `pci_switch_port` non-zero was removed from code so this suggestion of mine no longer applies?

Reply:
Ah okay, so the check is still there though in a different commit.  I'd prefer this limitation to be documented in `configuration-qemu-runtime-rs.toml.in` for the sake of our users, to avoid confusing them.